cob/cli/docker_cli.py

```diff
@@ -411,19 +411,32 @@ def _generate_compose_file_from_image(image_name, *, http_port=None):
 
 @docker.command(name='deploy', help='Deploys an dockerized cob app image to the local systemd-based machine')
 @click.option('--force', is_flag=True, default=False)
+@click.option('--compose-override', 'compose_override', default=None)
 @click.argument('image_name')
-def deploy_image(image_name, force):
+def deploy_image(image_name, force, compose_override):
     click.echo(f'Obtaining project information for {image_name}...')
     project_name = _get_project_name_from_image(image_name)
     unit_template = load_template('systemd_unit')
     filename = Path('/etc/systemd/system') / f'{project_name}-docker.service'
+
+    if compose_override:
+        docker_compose_override_file = Path(compose_override)
+        click.echo(f'Checking the existance of a docker compose override file under project root {docker_compose_override_file}')
+        if docker_compose_override_file.exists():
+            add_override = f'-o {docker_compose_override_file}'
+        else:
+            click.echo(f'File {docker_compose_override_file} does not exist')
+            raise click.Abort()
```

Comment:
Won't it look better to put the message in the exception constructor? I think in that case it is printed red etc.